frontend/src/api.js

```diff
@@ -428,12 +428,21 @@ class Api {
       subprojectId
     });
 
-  closeWorkflowItem = (projectId, subprojectId, workflowitemId) =>
+  closeWorkflowItem = (projectId, subprojectId, workflowitemId,rejectReason) => {
```

Comment:
change to `closeWorkflowItem = (projectId, subprojectId, workflowitemId,rejectReason = "")` to provide a default value